### ISSUE TITLE
Is uploading a movie currently unsupported?

### ISSUE THREAD

**Issue by untruenorth** [scrool/xled#4]
This may only be a documentation issue.  The private protocol documentation sets out the movie format, and a few words about how it's uploaded (https://xled.readthedocs.io/en/latest/protocol_details.html#upload-full-movie-led-effect), but I can't see the ability to actually upload it in the Python xled package. Nor does the API reference make this very clear.

Am I missing something (I know very little python) or is this not implemented yet? 

Thanks very much for your work on this so far though - it's appreciated.

**Comment by scrool** [comment]
You are right that python code doesn't correspond to documentation. That's because I have documented what firmware responds to. Then I have implemented library & CLI to talk with the firmware in more pythonic and linux-like way so I can do basic operations. Implementation definitely isn't complete.

Regarding movies I have found out there are other projects that controls home-made christmas lights. I was thinking about creating adapter to connect their API to this one. But I haven't finished that part yet.
